src/components/form/form.types.ts

```diff
@@ -104,3 +104,73 @@ export interface FormInfo {
      */
     name?: string;
 }
+
+/**
+ * Lime elements specific options that can be specified under the `lime` key in
+ * a schema, e.g.
+ *
+ * ```ts
+ * const schema = {
+ *     type: 'object',
+ *     lime: {
+ *         collapsible: true,
+ *     },
+ * };
+ * ```
+ */
+export interface LimeSchemaOptions {
```

Comment:
Not for this PR, but:

It's *really* nice to have this in the docs, but it's kind of hard to find, since the `schema` property is just typed as `{ [key: string]: any; id?: string; }`. Perhaps we can either expand that typing a little in the future, or just add some text with links to helpful "inner" typings… 🙂